packages/eslint-plugin/docs/rules/explicit-member-accessibility.md

```diff
@@ -75,6 +74,8 @@ A possible configuration could be:
 }
 ```
 
+Note the above is an example of a possible configuration you could use - it is not the default configuration.
```

Comment:
```suggestion
Note: This is an example of a possible configuration you could use - it is not the default configuration.
```
what do you think about something like this? or
```suggestion
Note: The above code is an example of a possible configuration you could use - it is not the default configuration.
```

I'm unsure if we have to say `it is not the default configuration.`, i think it is better to add clear note to:
`const defaultOptions: Options = {` adding this here.


----------------

i'm not that good at grammar

Reply:
TBH I don't think you even really need the clause, because the `Options` section at the start  has  `const defaultOptions: Options`.

https://github.com/typescript-eslint/typescript-eslint/blob/aee723813ec47ccac0a165cf1bc9674f6257b609/packages/eslint-plugin/docs/rules/explicit-member-accessibility.md#options

But I figured there's no harm in leaving it in.

Reply:
TBH I don't think you even really need the clause, because the `Options` section at the start  has  `const defaultOptions: Options`.

https://github.com/typescript-eslint/typescript-eslint/blob/aee723813ec47ccac0a165cf1bc9674f6257b609/packages/eslint-plugin/docs/rules/explicit-member-accessibility.md#options

But I figured there's no harm in leaving it in.

Reply:
I was gunning to fix the grammatical structure, rather than the actual content just b/c it wasn't *wrong*...

But yes to me that section feels a bit like a sore thumb, and ideally can be improved 🤷‍♂